packages/cli/src/commands/setup/deploy/providers/render.js

```diff
@@ -0,0 +1,77 @@
+import fs from 'fs'
+import path from 'path'
+
+import { getPaths } from 'src/lib'
+
+const RENDER_YAML = `services:
+- type: web
+  name: redwood-api
+  env: node
+  plan: starter
+  buildCommand: yarn && yarn rw prisma migrate deploy && yarn rw build api
+  startCommand: cd api && yarn api-server --functions ./dist/functions
+  envVars:
+  - key: DATABASE_URL
+    fromDatabase:
+      name: redwood-db
+      property: connectionString
+  - key: NODE_VERSION
+    value: 14.16.0
+
+- type: web
+  name: redwood-web
+  env: static
+  buildCommand: yarn && yarn rw build web
+  staticPublishPath: ./web/dist
+  envVars:
+  - key: API_PROXY_HOST
+    fromService:
+      name: redwood-api
+      type: web
+      property: host
+  routes:
+  - type: rewrite
+    source: /*
+    destination: /index.html
+
+databases:
+- name: redwood-db
+`
+const RENDER_HEALTH_CHECK = `
+export const handler = async () => {
+  return {
+    statusCode: 200,
+  }
+}
+`
+const NODE_VERSION = `14.16.0`
```

Comment:
How does Render handle Node.js version? For Redwood:
- each project installs with an `.nvmrc` file with contents `.lts/*`
- ideally, the versions shouldn't be statically pinned to a specific version unless the user specifically needs to do this for a given project

Reply:
We recommend using environment variables: https://render.com/docs/node-version

Reply:
The env var value can be a major version or range as well, like `14` or `<15`.

Reply:
Got it. Because a Redwood project already installs with 1) package.json Node.js Engine setting and 2) .nvmrc file, I would _not_ recommend adding a third possible Node version config option. 'Cause confusion. 

To discuss, but I'd recommend leaving things as is (because you can use Engine), but providing a "Next Step" in the output. Something like, "Render uses your Node version from Package.json. Learn more at this link. Also, we see you have a .nvmrc file — you should probably delete it."

Reply:
> We recommend using environment variables: https://render.com/docs/node-version

I am using the environment variables for node-version. The line tagged here was to also populate `.node-version` in the root. But that would is redundant.

Reply:
`engines` works well for now.

Reply:
resolved in latest per discussion